tests/models/bark/test_processor_bark.py

```diff
@@ -55,6 +55,15 @@ def test_save_load_pretrained_additional_features(self):
             pretrained_processor_name_or_path=self.checkpoint,
             speaker_embeddings_dict_path=self.speaker_embeddings_dict_path,
         )
+        """
+        TODO (ebezzam) not all speaker embedding are properly downloaded.
+        My hypothesis: there are many files (~700 speaker embeddings) and some fail to download (not the same at different first runs)
+        https://github.com/huggingface/transformers/blob/967045082faaaaf3d653bfe665080fd746b2bb60/src/transformers/models/bark/processing_bark.py#L89
+        https://github.com/huggingface/transformers/blob/967045082faaaaf3d653bfe665080fd746b2bb60/src/transformers/models/bark/processing_bark.py#L188
+        
+        So for testing purposes, we will remove the unavailable speaker embeddings before saving.
+        """
+        processor._verify_speaker_embeddings(remove_unavailable=True)
```

Comment:
TLDR need to remove speaker embedding which couldn't download properly. Maybe because too many? (~700)

Reply:
Format nit: let's use single-line comments (`#...`), and add an HF team member (@eustlb ?) in the TODO

Reply:
sure! and I'm a new HF team member working with @eustlb 😉